dapp-heroku/deploy-to-heroku.md

```diff
@@ -0,0 +1,110 @@
+# Deploying Your App to Heroku
+
+Earlier in the series, we deployed our Bounties.sol smart contract using Truffle, and added a react.js front end to interact with the contract through a web browser. In this tutorial we will deploy our front-end application to [Heroku](https://www.heroku.com).
+
+This tutorial uses the source code from the tutorial, [Truffle: Adding a frontend with react box](https://github.com/kauri-io/kauri-fullstack-dapp-tutorial-series/tree/master/truffle-react-box-frontend). You will have to add a Procfile to the project, but no other changes are required.
+
+## Heroku
+
+Heroku is a Platform-as-a-Service (PaaS) that enables developers to quickly build, deploy, and scale web applications. We'll use Heroku to quickly deploy our application and make it accessible to the public.
+
+## Prerequisites
+
+You'll need to configure the application, as defined in an earlier tutorial, [Truffle: Adding a frontend with react box](https://kauri.io/article/86903f66d39d4379a2e70bd583700ecf/v14/truffle:-adding-a-frontend-with-react-box).
+
+Additionally, you'll need to install the Heroku CLI.
+
+More information on the Heroku CLI can be found [here](https://devcenter.heroku.com/articles/heroku-cli).
+
+The Heroku CLI is available through Homebrew:
```

Comment:
@crawfordleeds Could you also add installation instructions (or links) for other platforms?

Reply:
Yes absolutely. Update coming shortly.

Reply:
Hey Chris, just pushed a change where I added the alternative command-line installation process for Ubuntu with Snap. In the existing link, there are a few alternative methods, primarily through an installer. I didn't want to put the links from those directly in this piece just in case Heroku changes those links down the road. Let me know if you think this is okay.